### PR TITLE
docs(readme): reorganize the en readme structure for better readability

### PR DESCRIPTION
**Pull request by yamasite**
If this change looks good, we can also update the cn readme per this structure.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,87 +25,81 @@
...
+    -   Support periodic rooms
+-   Classroom recording and replaying
+    -   Whiteboard replaying
+    -   Cloud recording for video and audio
+    -   Messaging replaying
+-   Cloud Storage for multi-media courseware
+-   Screen sharing
 
-## Related Projects
+## Quickstart
```
Comment (BlackHole1) [inline README.md:47]: QuickStart
or
Quick Start

**Review by BlackHole1 — CHANGES_REQUESTED** [review]
Look great to me. thank you for your  PR!

I'll review it more carefully tomorrow, until then I'll take a nap

**Comment by BlackHole1** [comment]
cc: @netless-io/flat @levi-oh

File: README.md (hunk 1 above)
Comment (LitoMore) [reply README.md:47]: `Quickstart` is the correct spell.

- https://developers.google.com/docs/api/quickstart/js
- https://docs.github.com/en/get-started/quickstart

Let add it to the cspell dictionary.

File: README.md [hunk 2]
```diff
@@ -25,87 +25,81 @@
...
-- [Start using Flat Web][flat-web]
-- [Download artifact][flat-homepage]
-- [Flat Components Storybook][flat-storybook]
+-   Real-time interaction
+    -   Multifunctional interactive whiteboard
+    -   Real-time video/audio chat (RTC)
+    -   Real-time messaging (RTM)
+-   Login via
+    -   WeChat
+    -   GitHub
```
Comment (LitoMore) [inline README.md]: GitHub is more international.

```suggestion
    -   GitHub
    -   WeChat
```

File: README.md (hunk 1 above)
Comment (BlackHole1) [reply README.md:47]: @yamasite 
please add `Quickstart` to https://github.com/netless-io/flat/blob/d6c668ab4fdd81b0662beee5b668d0a39604d058/cspell.config.js#L18-L19

**Review by hyrious — DISMISSED** [review]

**Comment by BlackHole1** [comment]
@yamasite can add `cn` readme.

**Review by LitoMore — CHANGES_REQUESTED** [review]
The https://github.com/netless-io/flat/blob/main/docs/readme/README-zh.md needs to be updated.

**Review by BlackHole1 — DISMISSED** [review]
LGTM

File: docs/readme/README-zh.md [hunk 3]
```diff
@@ -24,89 +24,84 @@
...
-- [快速体验 Flat Web][flat-web]
-- [应用 下载地址][flat-homepage]
-- [Flat 组件库 Storybook][flat-storybook]
+-   实时交互
+    -   多功能互动白板
+    -   实时音视频（RTC）通讯
+    -   即时消息（RTM）聊天
+-   帐户系统
+    -   微信登陆
+    -   GitHub 登陆
```
Comment (LitoMore) [inline docs/readme/README-zh.md]:
```suggestion
    -   GitHub 登录
    -   微信登录
```

Comment (LitoMore) [reply docs/readme/README-zh.md]: `登陆` is a typo. It should be `登录`.

**Review by LitoMore — DISMISSED** [review]

File: docs/readme/README-zh.md (hunk 3 above)
Comment (LitoMore) [reply docs/readme/README-zh.md]: BTW, I would prefer to call it `登录方式`.

```
- 登录方式
    - GitHub
    - 微信
```

**Comment by BlackHole1** [comment]
cc @LitoMore

**Pull request merged**